src/main/java/microsoft/exchange/webservices/data/autodiscover/AlternateMailbox.java

```diff
@@ -66,28 +76,31 @@ private AlternateMailbox() {
    * @return AlternateMailbox
    * @throws Exception the exception
    */
-  public static AlternateMailbox loadFromXml(EwsXmlReader reader)
+  public static AlternateMailbox loadFromXml(final EwsXmlReader reader)
       throws Exception {
-    AlternateMailbox altMailbox = new AlternateMailbox();
+    final AlternateMailbox altMailbox = new AlternateMailbox();
 
     do {
       reader.read();
 
       if (reader.getNodeType().getNodeType() == XmlNodeType.START_ELEMENT) {
-        if (reader.getLocalName()
-            .equalsIgnoreCase(XmlElementNames.Type)) {
+        if (reader.getLocalName().equalsIgnoreCase(XmlElementNames.Type)) {
           altMailbox.setType(reader.readElementValue(String.class));
-        } else if (reader.getLocalName().equalsIgnoreCase(
-            XmlElementNames.DisplayName)) {
-          altMailbox.setDisplayName(reader
-              .readElementValue(String.class));
-        } else if (reader.getLocalName().equalsIgnoreCase(
-            XmlElementNames.LegacyDN)) {
-          altMailbox.setLegacyDN(reader
-              .readElementValue(String.class));
-        } else if (reader.getLocalName().equalsIgnoreCase(
-            XmlElementNames.Server)) {
+        } else if (reader.getLocalName()
```

Comment:
If this is not the format as you expect please provide a different configuration for eclipse than GoogleStyle